es6-promises/main.js

```diff
@@ -4,8 +4,7 @@ const user = takeAChance('Justin');
 
 user.then((value) => {
   console.log(value);
-},
-(reason) => {
-  console.error(reason.message);
-}
-);
+})
```

Comment:
Indent this by 2 spaces.

I also suggest you eliminate the intermediate `user` variable.